src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -412,9 +412,20 @@ private static void resolveProject(AbstractBuild build, SubProjectData subProjec
 			// don't trigger on this configuration
 		}
         return Collections.emptyList();
-	}
-
+    }
 
+    private void reportSchedulingError(@Nonnull Run<?, ?> run, @Nonnull Job<?, ?> jobToTrigger, @Nonnull BuildListener listener) {
+        // Do not print details to Build Listener, they have been reported previously in #canTriggerProject()
+        listener.error("Skipping " + jobToTrigger.getFullName() + "...");
+        if (LOGGER.isLoggable(Level.CONFIG)) {
+            String message = String.format("Cannot schedule project %s. Job type is not parameterized, "
+                + "or there is no Item/Build permission for the current authentication %s. "
```

Comment:
As I have mentioned previously, `Item/Build` is nonexistent. The Java constant is `Item.BUILD`. The permission as displayed in the UI is `Job/Build`.